src/rebar_file_utils.erl

```diff
@@ -91,7 +96,9 @@ rm_rf(Target) ->
             ok = delete_each(Files),
             ok = delete_each_dir_win32(Dirs),
             ok
-    end.
+    end;
+rm_rf(Target) when is_binary(Target) ->
+    rm_rf(binary_to_list(Target)).
```

Comment:
Given the type spec of this function extects `string()` I'd say we should rather convert before calling it, or at the very least update the spec.

In any case, direct conversion of binary -> list like that will break on systems where the system path is encoded in say UTF8 and special characters are used. Proper decoding would require using `unicode:characters_to_list` in such cases. I don't know how good of a job we can do without knowing what we get (latin1 can look the same as utf8 but mean different things..)